### DISCUSSION TITLE
CMake Project Configuration for Visual Studio Code

### DISCUSSION THREAD

**Discussion by stdeguzman** [ANLAB-KAIST/KENSv3#30] [Project Q&A 1. Application Layer (Socket)] [answered]
I'm trying to setup my Visual Studio Code for the project, but when I configure my CMake project, I get this error. Could someone help me on fixing this? Thank you.
![VSCode](https://user-images.githubusercontent.com/65854216/159108154-f0c38ddc-dd30-4191-b988-62f92aedcede.png)

**Comment by Pusnow** [comment, marked as answer]
Hi, currently we do not support Windows native environment. Try WSL instructions.

**Reply by stdeguzman** [reply]
May I ask where the WSL instructions are? I'm sorry, I'm getting confused with all the links.

**Reply by Pusnow** [reply]
The instruction is here: https://github.com/ANLAB-KAIST/KENSv3/wiki/Getting-Started:-CLI-(Windows-WSL) .

1. Install WSL2
2. Follow Linux's CLI instruction in WSL.
3. Install Remote - WSL package: https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl

Then you can use WSL in VSCode.
